packages/relay/src/lib/eth.ts

```diff
@@ -688,24 +688,31 @@ export class EthImpl implements Eth {
     this.logger.trace(`${requestIdPrefix} getTransactionCount(address=${address}, blockNumOrTag=${blockNumOrTag})`);
     const blockNumber = await this.translateBlockTag(blockNumOrTag, requestId);
     if (blockNumber === 0) {
-      return '0x0';
-    } else {
-      try {
-        const result = await this.mirrorNodeClient.resolveEntityType(address, requestId);
-        if (result?.type === constants.TYPE_ACCOUNT) {
-            const accountInfo = await this.sdkClient.getAccountInfo(result?.entity.account, EthImpl.ethGetTransactionCount, requestId);
-            return EthImpl.numberTo0x(Number(accountInfo.ethereumNonce));
-        }
-        else if (result?.type === constants.TYPE_CONTRACT) {
-          return EthImpl.numberTo0x(1);
-        }
-      } catch (e: any) {
-        this.logger.error(e, `${requestIdPrefix} Error raised during getTransactionCount for address ${address}, block number or tag ${blockNumOrTag}`);
-        return predefined.INTERNAL_ERROR;
+      return EthImpl.zeroHex;
+    } else if (address && !blockNumOrTag) {
+      // get latest ethereumNonce
+      const mirrorAccount = await this.mirrorNodeClient.getAccount(address, requestId);
```

Comment:
I just want to confirm mirror node is now always up to date, not not up to 15 minutes behind anymore.

Reply:
Correct the `api/v1/accounts` was always up to data for account info but is now also up to data for balance info.
`api/v1/balance` is still on the 15 min behind schedule until mirror node updates it's logic